scripts/sum-up.py

```diff
@@ -0,0 +1,77 @@
+import os
+import subprocess
+
+OUTPUT_UBUNTU   = eval(os.environ.get('OUTPUT_UBUNTU', ''))
+OUTPUT_MACOS    = eval(os.environ.get('OUTPUT_MACOS', ''))
+OUTPUT_ALPINE   = eval(os.environ.get('OUTPUT_ALPINE', ''))
+OUTPUT_WINDOWS  = eval(os.environ.get('OUTPUT_WINDOWS', ''))
+OUTPUT_RV       = eval(os.environ.get('OUTPUT_RV', ''))
+
+with open(os.environ.get('GITHUB_STEP_SUMMARY'), 'w') as f:
+    any_file_looks_odd = False
+    for key in OUTPUT_UBUNTU:
+        assert all(key in d for d in [OUTPUT_UBUNTU, OUTPUT_MACOS, OUTPUT_ALPINE, OUTPUT_WINDOWS, OUTPUT_RV])
+        this_file_looks_odd = False
+        print(f'\n## {key}\n')
+        f.write(f'\n## **{key}**\n')
+
+        print(f'x86_64 Ubuntu 22.04')
```

Comment:
Using an array of tuple `(system_info, output)` may be better than hardcoding it.

Reply:
Fixed with https://github.com/OI-wiki/OI-wiki/pull/5821/commits/2d24818212d54be8b9149c575ea1078ebb1597e5 and please check https://github.com/OI-wiki/OI-wiki/actions/runs/10717579136?pr=5821#summary-29717785913 for new log.